virt/support/virt-troubleshooting.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+[id="virt-troubleshooting"]
+= Troubleshooting
+include::_attributes/common-attributes.adoc[]
+:context: virt-troubleshooting
+
+toc::[]
+
+[id="events_{context}"]
+== Events
+
+xref:../../nodes/clusters/nodes-containers-events.adoc#nodes-containers-events[{product-title} events] are records of important life-cycle information and are useful for monitoring and troubleshooting resource issues.
```

Comment:
Might be helpful to have a small introductory phrase here before the bulleted list, such as "You can get information about the following events:"